src/components/experiment-tracking/time-series/time-series.js

```diff
@@ -7,13 +7,16 @@ import { getTooltipPosition } from '../tooltip/get-tooltip-position';
 import * as d3 from 'd3';
 
 import './time-series.css';
+import { usePrevious } from '../../../utils/hooks';
 
 export const TimeSeries = ({ metricsData, selectedRuns }) => {
   const [width, setWidth] = useState(0);
   const [showTooltip, setShowTooltip] = useState(tooltipDefaultProps);
 
   const { hoveredElementId, setHoveredElementId } =
     useContext(HoverStateContext);
+  const previousselectedRuns = usePrevious(selectedRuns);
+  const [rangeSelection, setRangeSelection] = useState();
```

Comment:
Nit: please move this up to where the other `useState` hooks are. Also, this should probably have a default value.